client/src/entrypoints/admin/comments.js

```diff
@@ -353,7 +353,7 @@ window.comments = (() => {
     commentCounter.className =
       '-w-mr-3 w-py-0.5 w-px-[0.325rem] w-translate-y-[-8px] w-translate-x-[-4px] w-text-[0.5625rem] w-font-bold w-bg-secondary-100 w-text-white w-border w-border-white w-rounded-[1rem]';
     commentToggle.className =
-      'w-h-slim-header w-bg-transparent w-box-border w-py-3 w-px-3 w-flex w-justify-center w-items-center w-outline-offset-inside w-text-grey-400 w-transition hover:w-transform hover:w-scale-110 hover:w-text-primary focus:w-text-primary';
+      'w-h-slim-header w-bg-transparent w-box-border w-py-3 w-px-3 w-flex w-justify-center w-items-center w-outline-offset-inside w-text-grey-400 w-transition hover:w-transform hover:w-scale-110 hover:w-text-primary focus:w-text-primary expanded:w-text-primary';
```

Comment:
Is there any way to avoid the new tailwind variant to achieve this goal? Maybe passing in a className or reading the expanded state when preparing this class?

Reply:
We could add a default class for these sidebar toggles then select aria-expanded and do the same as the variant does but I'm curious why we would want to make this more complex than it needs to be?

**Note:** We also need to change styles based on aria-expanded in the html templates `side_panel_toggles.html` for these buttons

Reply:
I guess I don't really understand the direction we are heading with Tailwind and when we should be using it or not using it. 

But for now I think this makes sense I guess with the existing code being changed.

Reply:
Definitely understandable, I believe we might be swapping out utility styling like this in the future in favour of using the `theme()` function but for this PR I had intended to make minimal adjustments to avoid too many regressions 😰